macos/FluentUI/Core/Colors.swift

```diff
@@ -247,6 +247,40 @@ public final class Colors: NSObject {
 			}
 		}
 	}
+	
+	// MARK: Primary
+	
+	@objc public var primary: NSColor = Colors.Palette.communicationBlue.color {
```

Comment:
similar to iOS, i think it should be static class variable so that you don't need to create instance of MSFColors